mk/uvmt/vsim.mk

```diff
@@ -177,8 +176,25 @@ VSIM_UVM_ARGS      = +incdir+$(UVM_HOME)/src $(UVM_HOME)/src/uvm_pkg.sv
 #  endif
 #endif
 
-
-ifeq ($(call IS_YES,$(USE_ISS)),YES)
+GCC_PATH			= $(HOME)/opt/gcc-11.4.0/bin/gcc
+
+VSIM_FLAGS        	+= -dpicpppath $(GCC_PATH)
+VSIM_FLAGS 			+= -noautoldlibpath
+
+# This must currently always be exported, since the reference_model_wrap is used in uvmt_cv32e40s_tb.sv
+# TODO: Make a "dummy" reference model, that is loaded when it is disabled, like ImperasDV. 
+# Or find another solution to handle swapping different ISSs
+export FILE_LIST_RM      ?= -f $(DV_UVMT_PATH)/reference_model.flist
+
+ifeq ($(call IS_YES,$(USE_RM)),YES)
```

Comment:
I'm struggling to see what is completely new vs what is copied from "master" and will disappear from this diff after the other PR is merged.

Reply:
All of these changes are new, and not a part of master